src/azul/deployment.py

```diff
@@ -6,38 +6,38 @@
 import botocore.session
 
 
-def memoized_property(f):
+def memorized_property(f):
```

Comment:
I don't think this is a typo.  I think it means memoization as in caching

Reply:
Yup.